addons/account/models/company.py

```diff
@@ -561,3 +561,16 @@ def get_chart_of_accounts_or_fail(self):
                 "Please go to Account Configuration and select or install a fiscal localization.")
             raise RedirectWarning(msg, action.id, _("Go to the configuration panel"))
         return account
+
+    def get_fiscal_country(self):
+        """ Returns the country to be used for this company's tax report.
+        By default, it'll be the one from the address; but a config parameter
+        may be used for each company to customize that behavior.
+        """
+        self.ensure_one()
+
+        fiscal_country_key = 'account_fiscal_country_%s' % self.id
+        forced_country_code = self.env['ir.config_parameter'].get_param(fiscal_country_key)
```

Comment:
Hello, there is a missing `sudo` in here, the CRUD operations can only be done if you have settings access rights.

And I think this fix should have a test to make sure it is not broken later.

Regards

Reply:
Hi @tbe-odoo . Done here https://github.com/odoo/odoo/pull/53222 . I don't think it's worth adding a test for this; I have a task for master to handle this in a less "hacky" way anyway, so this patch won't live long ;)

Reply:
Thanks a lot :D